replies.go

```diff
@@ -118,6 +118,11 @@ var replies = []Reply{
 		message:   "Like the " + Weeb.Mention() + "s?",
 		onlyRoles: []Role{Weeb},
 	},
+	{
+		pattern:   `nigger|cunt`,
+		message:   "Please refrain from using strong language.",
+		excludeRoles: []Role{Support},[]Role{Oldfriend},[]Role{Trooper},
+	},
```

Comment:
Not sure if we need this to be automated, i could see this leading to people spamming these words just to trigger the bot response.

Also, excluding `Support` is redundant since staff are only replied to if they @mentioned the bot.
Not sure ottomh if `Oldfriend` or `Trooper` are declared in the `consts` file or not either.

Also the syntax is incorrect here. Rather than `[]Role{Support},[]Role{Oldfriend},[]Role{Trooper},` it should be `[]Role{Support, Oldfriend, Trooper},`. If you're able to, it would be much easier if you made changes like this in an IDE (or at least manually ran `gofmt` locally) to ensure things are both syntactically correct and formatted correctly.